packages/client/components/MeetingSidebarTeamMemberStageItems.tsx

```diff
@@ -36,23 +36,25 @@ interface Props {
 const MeetingSidebarTeamMemberStageItems = (props: Props) => {
   const {gotoStageId, handleMenuClick, meeting, phaseType} = props
   const {facilitatorStageId, facilitatorUserId, localPhase, localStage, phases} = meeting
-  const sidebarPhase = phases.find((phase) => phase.phaseType === phaseType)!
+  const sidebarPhase = phases.find((phase) => phase.phaseType === phaseType)
   const localStageId = (localStage && localStage.id) || ''
   const gotoStage = (teamMemberId) => () => {
-    const teamMemberStage = sidebarPhase.stages.find((stage) => stage.teamMemberId === teamMemberId)
+    const teamMemberStage =
+      sidebarPhase && sidebarPhase.stages.find((stage) => stage.teamMemberId === teamMemberId)
```

Comment:
can always write `sidebarPhase?.stages` to clean it up a bit. not a show stopper